### PR TITLE
Add console button tooltip and document link when button disabled

### PR DESCRIPTION
**Pull request by a110605**
### Summary
- Add tooltip and external link icon if no consoleUrl is set.
- Extract all harvester doc links into a config `doc-links.js` and auto construct the doc link with the harvester version in package.json.
    - Discussed conclusion with @jillian-maroket  (https://github.com/harvester/harvester/issues/6403)

#### PR Checklist
- Is this a multi-tenancy feature/bug?
    - [ ] Yes, the relevant RBAC changes are at:
- Do we need to backport changes to the [old Rancher UI](https://github.com/rancher/u), such as RKE1?
    - [ ] Yes, the relevant PR is at:
- Are backend engineers aware of UI changes?
    - [x] Yes, the backend owner is: @WebberHuang1118 

Related Issue # 
- https://github.com/harvester/harvester/issues/5679
- https://github.com/harvester/harvester/issues/6403

### Screenshot/Video

https://github.com/user-attachments/assets/969855af-ec23-4f3c-912c-59971a925e8f

### CODE REVIEW

File: pkg/harvester/config/doc-links.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+export const DOC_LINKS = { CONSOLE_URL: 'https://docs.harvesterhci.io/v1.4/host/' };
```
Comment (a110605) [inline pkg/harvester/config/doc-links.js:2]: Hi @WebberHuang1118 , we can update this link once we have a document block describe how to setup console URL.

Comment (bk201) [reply pkg/harvester/config/doc-links.js:2]: Is there a way not to hard-code the version? (and determine from current running branches).

Comment (bk201) [reply pkg/harvester/config/doc-links.js]: Sounds great, as long as the version in the package.json file is bumped correctly on each place.

File: pkg/harvester/components/UpgradeInfo.vue [hunk 2]
```diff
@@ -26,15 +31,14 @@ export default {
     <Banner color="warning">
       <div>
         <strong>{{ t('harvester.upgradePage.upgradeInfo.warning') }}:</strong>
-        <p v-clean-html="t('harvester.upgradePage.upgradeInfo.doc', {}, true)" class="mb-5">
-        </p>
+        <p v-clean-html="t('harvester.upgradePage.upgradeInfo.doc', {url: upgradeLink}, true)" class="mb-5"></p>
```
Comment (a110605) [inline pkg/harvester/components/UpgradeInfo.vue:34]: `target="_blank"` is ignore when giving translation key via `v-clean-html`. (not open link in new tab)
It's resolved in rancher but not harvester dashboard..

PR: https://github.com/rancher/dashboard/pull/9928

File: pkg/harvester/l10n/en-us.yaml [hunk 3]
```diff
@@ -307,6 +307,7 @@ harvester:
     console: Console
     label: Hosts
     inconsistentIP: "Host IP is inconsistent, current IP: { currentIP }, initial IP: { initIP }"
+    noConsoleUrl: 'Console URL is empty'
```
Comment (jillian-maroket) [inline pkg/harvester/l10n/en-us.yaml:310]:
```suggestion
    noConsoleUrl: 'Console URL not specified.'
```

**Review by jillian-maroket — COMMENTED** [review]
Thanks for creating the mapping file. We will need to update the links once the product docs are available on the [SUSE doc site](https://documentation.suse.com/).

Please apply the suggested string change.

File: pkg/harvester/l10n/en-us.yaml (hunk 3 above)
Comment (a110605) [reply pkg/harvester/l10n/en-us.yaml]: Done modified. 
We can easily update the harvester docs in `config/doc-links.js`. Just let us know the new product docs is online available.

**Review by jillian-maroket — APPROVED** [review]

**Review by torchiaf — APPROVED** [review]
LGTM

**Comment by a110605** [comment]
@mergify backport release-harvester-v1.4

**Pull request merged**